registration.php

```diff
@@ -20,7 +22,7 @@ function registerForOfflineEvent($event){
 
         // already registered
         if(strpos(file_get_contents($file), $mail) !== false || strpos(file_get_contents($file), $phone) !== false){
-            echo "<div class='block info'>Du bist zu dieser Veranstaltung bereits angemeldet.</div>";
+            echo "<div class='block warning'>Du bist zu dieser Veranstaltung bereits angemeldet.</div>";
```

Comment:
Does this work? There is indeed already a definition for `error`:
```
css/style.css:.error {
css/style.css-    color: red;
css/style.css-    border: 1px solid red;
css/style.css-    border-radius: 2px;
css/style.css-    margin: 5px 0;
css/style.css-    padding: 2px;
css/style.css-}
```

But I didn't find a class definition for `warning`.